operator/gitops/argocd/pipeline-service/kustomization.yaml

```diff
@@ -4,7 +4,6 @@ kind: Kustomization
 
 resources:
   - openshift-pipelines
-  - pipelines-as-code
   - tekton-chains
```

Comment:
I would expect chains to be installed by the operator as well since it's GA.

Reply:
+1
that was our understanding back during our last round of feature refinement, as installing it from the operator, aside from "reduced duplication", gets us "use RH based images" etc.

Reply:
in fact the test failure implies to me "let's get the chains installed via 1.10 to get versions correct:
```
- Public key: Error from server (BadRequest): error when creating "/source/operator/test/manifests/test/tekton-chains/public-key.yaml": admission webhook "webhook.pipeline.tekton.dev" denied the request: mutation failed: cannot decode incoming new object: json: unknown field "serviceAccountName"
command terminated with exit code 1
```